### PR TITLE
Update deploy scripts to reflect current deploy strategy

### PR DESCRIPTION
**Pull request by btoll**
## Summary

The deployment stage is currently being run manually and this PR updates or replaces the older deploy scripts that were not being used.

Bear in mind that the deb deploy script will be replaced soon-ish by a container that already has the packages needed to create additional snapshots to deploy, so I wouldn't focus too much on reviewing this particular script.

## Test Plan

`mule -f package-deploy package-deploy-rpm`

The deb package deploy scripts are run from the `deploy/deb` directory.

### CODE REVIEW

File: scripts/release/mule/deploy/deb/deploy.sh [hunk 1]
```diff
@@ -2,78 +2,52 @@
...
-ARCH_TYPE=$(./scripts/archtype.sh)
-OS_TYPE=$(./scripts/ostype.sh)
-VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
-BRANCH=${BRANCH:-$(git rev-parse --abbrev-ref HEAD)}
-CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
-PKG_DIR="./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
-SIGNING_KEY_ADDR=dev@algorand.com
+if [ $# -lt 3 ]
+then
+    echo "Usage: $0 PROJECT CHANNEL VERSION"
```
Comment (egieseke) [inline scripts/release/mule/deploy/deb/deploy.sh:7]: Should this usage info include the optional SNAPSHOT parameter?

File: scripts/release/mule/deploy/deb/deploy.sh [hunk 2]
```diff
@@ -2,78 +2,52 @@
...
 
-if ! $USE_CACHE
+if [ "$PROJECT" = indexer ]
 then
-    export ARCH_BIT
-    export ARCH_TYPE
-    export CHANNEL
-    export OS_TYPE
-    export VERSION
+    echo abort
```
Comment (egieseke) [inline scripts/release/mule/deploy/deb/deploy.sh:20]: Consider including an explanation for abort.

File: scripts/release/mule/deploy/deb/deploy.sh [hunk 3]
```diff
@@ -2,78 +2,52 @@
...
-}
-EOF
+if [ -z "$SNAPSHOT" ]
+then
+    SNAPSHOT="$REPO-$VERSION"
+fi
 
-DEBS_DIR="$HOME/packages/deb/$CHANNEL"
-DEB="algorand_${CHANNEL}_linux-amd64_${VERSION}.deb"
+echo -e "PROJECT $PROJECT\nREPO $REPO\nSNAPSHOT $SNAPSHOT\nFILENAME_PREFIX $FILENAME_PREFIX"
```
Comment (egieseke) [inline scripts/release/mule/deploy/deb/deploy.sh:42]: Include channel and version in info message?

File: scripts/release/mule/deploy/deb/deploy.sh [hunk 4]
```diff
@@ -2,78 +2,52 @@
...
 
-cp "$PKG_DIR/$DEB" "$DEBS_DIR"
+aws s3 cp "s3://algorand-staging/releases/$CHANNEL/$VERSION/${FILENAME_PREFIX}_${CHANNEL}_linux-amd64_${VERSION}.deb" "$PKG_DIR"
 
-SNAPSHOT="${CHANNEL}-${VERSION}"
-aptly repo create -distribution="$CHANNEL" -component=main algorand
-aptly repo add algorand "$DEBS_DIR"/*.deb
-aptly snapshot create "$SNAPSHOT" from repo algorand
-aptly publish snapshot -gpg-key="$SIGNING_KEY_ADDR" -origin=Algorand -label=Algorand "$SNAPSHOT" "s3:algorand-releases:"
+#aptly repo create -distribution="devtools" -architectures="amd64" -component="main" -comment="devtools-stable" algorand-devtools
```
Comment (egieseke) [inline scripts/release/mule/deploy/deb/deploy.sh:46]: Can these commented lines be removed?

**Comment by btoll** [comment]
I just removed the deploy.sh.  The intent of this PR is to get the docker and release page scripts into the project.

The deb deploy process will be improved this sprint.  This PR is just a step along the way towards a better pipeline.

**Review by bricerisingalgorand — APPROVED** [review]

**Review by onetechnical — DISMISSED** [review]

**Review by onetechnical — APPROVED** [review]

**Review by bricerisingalgorand — APPROVED** [review]

**Pull request merged**